app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -353,6 +357,29 @@ class HomeFragment : Fragment(), AccountObserver {
                     share(session.url)
                 }
             }
+            is TabAction.PauseMedia -> {
+                // TODO: This will be replaced with a simple call to `MediaStateMachine.pause` once AC exposes
+
+                sessionManager.findSessionById(action.sessionId)?.let { session ->
+                    for (media in session.media) {
+                        // TODO: Should I loop the whole list...?
+                        if (media.state == Media.State.PAUSED) {
+                            media.controller.play()
+                        } else {
+                            media.controller.pause()
+                        }
+                    }
+                }
+            }
+            is TabAction.PlayMedia -> {
+                // TODO: This will be replaced with a simple call to `MediaStateMachine.play` once AC exposes
```

Comment:
Will be exposed here: https://github.com/mozilla-mobile/android-components/pull/4351